src/graphql/event/dns.rs

```diff
@@ -144,6 +149,26 @@ impl DnsCovertChannel {
             .as_ref()
             .map(|scores| scores.iter().map(Into::into).collect::<Vec<TriageScore>>())
     }
+
+    async fn level(&self) -> ThreatLevel {
+        Self::get_threat_level()
```

Comment:
I don't think calling this method is required. It is simpler to return `ThreatLevel::Medium` directly, which makes the `EventThreatLevel` trait unnecessary.